### ISSUE TITLE
Upgrade core-js v3

### ISSUE THREAD

**Issue by galtalmor** [storybookjs/addon-smart-knobs#52]
Trying to use smart-knobs with core-js 3.1.4 (currently latest) results in the following error:
```
ERROR in ./node_modules/storybook-addon-smart-knobs/dist/index.js
Module not found: Error: Cannot find module 'core-js/modules/es6.regexp.replace'
 @ ./node_modules/storybook-addon-smart-knobs/dist/index.js 36:0-45
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true

ERROR in ./node_modules/storybook-addon-smart-knobs/dist/index.js
Module not found: Error: Cannot find module 'core-js/modules/web.dom.iterable'
 @ ./node_modules/storybook-addon-smart-knobs/dist/index.js 28:0-43
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true
```

This is because of the following:

> Yeah as per their change log for v3.0,
> 
> * Move all  features from ES5, ES2015, ES2016, ES2017, ES2018 and ES2019 to one namespace for stable ES - it's available as core-js/es, all those features in modules folder has es. prefix.
> * Change prefix for ES proposals from es7. to esnext., they no longer available in core-js/es7, use core-js/stage/* instead of that.

**Comment by hakan-bilgin** [comment]
I got the same error.

**Comment by praneybehl** [comment]
Same issue here, for me I had to drop the version down to 2.6.5 to get it working

**Comment by ndelangen** [comment]
FIXED in 5.0.0 which I just released
